src/sqlfluff/cli/commands.py

```diff
@@ -240,7 +242,8 @@ def core_options(f: Callable) -> Callable:
         "--encoding",
         default="autodetect",
         help=(
-            "Specifiy encoding to use when reading and writing files. Defaults to autodetect."
+            "Specifiy encoding to use when reading and writing files. Defaults to "
```

Comment:
Noticed a pre-existing typo:
```suggestion
            "Specify encoding to use when reading and writing files. Defaults to "
```